src/utils/story.utils.js

```diff
@@ -1,25 +1,22 @@
-import { action } from '@storybook/addon-actions' // eslint-disable-line
+import { action } from '@storybook/addon-actions'
 
-const logAction = (actionTaken, componentName) => {
-  console.log(actionTaken, componentName) // eslint-disable-line
+const logEvent = (eventType, value) => {
+  console.log(eventType, value)
 }
 
-export const handleClick = (componentName) => {
-  logAction('Clicked: ', componentName)
-}
+export const handleClick = name => logEvent('Clicked: ', name)
+export const handlePolling = name => logEvent('Polling: ', name)
+export const handleAction = name => logEvent('Action Submitted: ', name)
 
-export const handlePolling = (componentName) => {
-  logAction('Polling: ', componentName)
-}
+export const MockedStore = (key, data) => ({
+  getState: () => ({ [key]: data }),
+  subscribe: () => 0,
+  dispatch: () => action('Dispatched Action'),
+})
 
-export const handleAction = (componentName) => {
-  logAction('Action Submitted: ', componentName)
+export default {
```

Comment:
We're exporting these twice now ;) 

- as an aside - I'd like to hear your thoughts on grouped export vs individual - I used to group, but I think individual exports make more sense in a bigger file because it's easy to see if an individual file is being exported (but downside it's hard to see the API of all available functions)

Reply:
@cameronolivier I think individual exports.

I sometimes like to have a default export and I guess you can then import the default easily, but I've removed it for this module now.

Reply:
ok cool :)